docs/source/upgrade-guide/index.md

```diff
@@ -39,6 +39,14 @@ This aligns Volto with the latests developments in the React ecosystem and opens
 - Transitions
 - New hooks `useId`, `useTransition`, `useDeferredValue`, `useSyncExternalStore`, and other hooks
 
+### `draftJS` dependency and `text`, `table`, and `hero` block removed
+
+The `text` block powered by `draftJS` library was deprecated in Volto 16, and announced that it will be removed from Volto in 18.
+The other two blocks powered by `draftJs` (`table` and `hero`).
```

Comment:
I think you left something off this sentence. Is this better?

```suggestion
The other two blocks powered by `Draft.js`, `table` and `hero` 🦗 🦗 .
```

Reply:
Done.